src/Microsoft.Sbom.Contracts/Contracts/Enums/ErrorType.cs

```diff
@@ -46,9 +46,15 @@ public enum ErrorType
     [EnumMember(Value = "Json serialization error")]
     JsonSerializationError = 5,
 
+    /// <summary>
+    /// Element does not meet compliance standard.
+    /// </summary>
+    [EnumMember(Value = "Compliance standard validation error")]
+    ComplianceStandardError = 6,
+
     /// <summary>
     /// An unknown error occured.
     /// </summary>
     [EnumMember(Value = "Other")]
-    Other = 6,
+    Other = 7,
```

Comment:
Strictly speaking, the return value can be treated as part of our interface, so changing the "Other" error isn't something that we want to do frequently. Enum's don't need to be sequential, so let's bump this to 255, which is the largest return value supported by batch scripting.